decisions/0006-program-slugs-in-publisher.rst

```diff
@@ -0,0 +1,32 @@
+6. Program Slugs in Publisher
+-------------------------------
+
+Status
+------
+
+Accepted (July 2023)
+
+Context
+-------
+
+Ability to update the slugs for programs is needed to be accessible to a group of users.
+These slugs are used in generating url for each type of programs.
+These program slugs will be programmatically generated, just like courses.
+In order to grant the ability to update slugs for programs, we're going to display programs on Publisher due to it's ease of access control.
```

Comment:
nit: Add a bit of generic context of why slugs are important in SEO. 

* generating url -> generating marketing url
* program slugs are not currently automatically generated
* Instead of using first-person pronouns, make the sentences third party (remove terms like "we're, we will")